net-analyzer/smokeping/smokeping-2.7.3.ebuild

```diff
@@ -142,6 +142,6 @@ src_install() {
 }
 
 pkg_postinst() {
-	chown smokeping:smokeping "${ROOT}/var/lib/${PN}"
-	chmod 755 "${ROOT}/var/lib/${PN}"
+	chown smokeping:smokeping "${EROOT}"/var/lib/${PN}
+	chmod 755 "${EROOT}"/var/lib/${PN}
```

Comment:
missing `die`?

Reply:
I've had people question die in postinst before and I never bothered thinking about it before, let's add it though, people deserve to know when things fail.